libbpf-rs/src/query.rs

```diff
@@ -498,29 +498,80 @@ gen_info_impl!(
 // TODO: Document members.
 #[allow(missing_docs)]
 pub struct BtfInfo {
-    pub btf: u64,
-    pub btf_size: u32,
+    pub name: String,
+    pub btf: Vec<u8>,
     pub id: u32,
 }
 
 impl BtfInfo {
-    fn from_uapi(_fd: i32, s: libbpf_sys::bpf_btf_info) -> Option<Self> {
-        Some(Self {
-            btf: s.btf,
-            btf_size: s.btf_size,
-            id: s.id,
-        })
+    fn load_from_fd(fd: i32) -> Option<Self> {
+        let mut item: libbpf_sys::bpf_btf_info = unsafe { std::mem::zeroed() };
```

Comment:
Nit: let's import `std` items we use at the top for the sake of consistency with existing code:
```suggestion
        let mut item: libbpf_sys::bpf_btf_info = unsafe { mem::zeroed() };
```